packages/unified-latex-to-pretext/libs/pre-conversion-subs/expand-user-defined-macros.ts

```diff
@@ -0,0 +1,49 @@
+import * as Ast from "@unified-latex/unified-latex-types";
```

Comment:
This is code from the previous PR, right? You need to merge in the `pretext` branch